Automattic/wp-calypso
# Create the ManageSitePluginsDialog and use it on the Manage Sites button

#### Proposed Changes

* Create the ManageSitePluginsDialog component with the Site List and its functions (activate, autoupdate, remove, settings, etc)
* Use it on the Plugin Details page

<img width="1150" alt="Screen Shot 2022-08-05 at 17 11 36" src="https://user-images.githubusercontent.com/5039531/183217325-9d6613d8-02db-4221-967b-96f8b3dc0f4f.png">


#### Testing Instructions

* Enable the feature flag `plugins/plugin-details-layout`*
* Go to a multi-site plugin page (`/plugins/{plugin_slug}`)
* Verify you see the `Manage sites` button
* Click on the button 
* You should see a section with all sites available (atomic), being the plugin installed or uninstalled.
* Test all the available options (activate, autoupdate, remove, settings, etc)

\* To enable this feature flag add `?flags=plugins/plugin-details-layout` to the end of your URL or past the following code on the developer tools console `document.cookie = 'flags=plugins/plugin-details-layout;max-age=1209600;path=/'`

#### Pre-merge Checklist






---

Fixes #63888
